server/apps/bindings.go

```diff
@@ -0,0 +1,65 @@
+package apps
```

Comment:
Resolve: the app's `/command/X` should translate into `/command/appCommand/X`

Reply:
I'm not sure what you mean by this. Are you saying there's an endpoint with that structure?

Reply:
https://mattermost.atlassian.net/browse/MM-30036